docs/walkthrough.md

```diff
@@ -193,6 +193,15 @@ kubectl config set-cluster service-catalog --server=http://$SVC_CAT_API_SERVER_I
 kubectl config set-context service-catalog --cluster=service-catalog
 ```
 
+Note: if you are using Google cloud or other cloud providers, you may need to
```

Comment:
I like the note here.  "Your cloud provider may require firewall rules to allow your traffic get in."

I think we should move the following details on particular cloud providers to a new section at the end, and link there from here.